homeassistant/components/eheimdigital/config_flow.py

```diff
@@ -0,0 +1,93 @@
+"""Config flow for EHEIM Digital."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from aiohttp import ClientError
+from eheimdigital.hub import EheimDigitalHub
+import voluptuous as vol
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import SOURCE_USER, ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import selector
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
+    {vol.Required(CONF_HOST, default="eheimdigital"): selector.TextSelector()}
```

Comment:
what's that for host

Reply:
The device registers as "eheimdigital" in the local DNS server, so entering something different is only needed if your local DNS isn't working.